site/src/patterns/side-scrolling-container.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Side scrolling container
+description: Side scrolling container pattern in the Atlas Design System
+template: standard
+---
+
+# Side scrolling container
+
+WIP
+
+```html
+<div class="scroll-horizontal scroll-snap-container display-flex padding-bottom-xs">
```

Comment:
Two ways to go here:

1. A new css file in /components scalled scroll-nap. Then keep, then rename this new documentation page to match. We would then want to link to original scroll page, from the newly created markdown file to enhance discoverability.
2. Retain the organization of scroll.scss, but move this example into the paired markdown file, scroll.md. 

I'm close the fence, though I may slightly favor the latter. The main thing is that the file names are synced and that scroll based documentation is easily discoverable.

Reply:
moved to components and under scroll.md